docker/bin/docker-go-echo.sh

```
@@ -5,17 +5,18 @@
 
 ARG="${1}"
 SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+PROJECT_NAME="auth-go-echo"
 cd "${SCRIPT_DIR}/../local/app-go-echo"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)